client/windows/CAdvmapInterface.cpp

```diff
@@ -470,6 +530,9 @@ CAdvMapInt::CAdvMapInt():
   spellBeingCasted(nullptr), position(int3(0, 0, 0)), selection(nullptr),
   updateScreen(false), anim(0), animValHitCount(0), heroAnim(0), heroAnimValHitCount(0),
 	activeMapPanel(nullptr), duringAITurn(false), scrollingDir(0), scrollingState(false)
+#ifdef VCMI_ANDROID
+	, swipeEnabled(settings["general"]["swipe"].Bool()), swipeMovementRequested(false), swipeTargetPosition(int3(-1,-1,-1))

```

Comment:
Okay, good point. Actually it is added in the config, but I didn't apply it to the repo (in android project, config folder is embedded inside the app).